geode-core/src/main/java/org/apache/geode/internal/net/SSLConfigurationFactory.java

```diff
@@ -168,7 +168,7 @@ private SSLConfig createSSLConfig(final DistributionConfig distributionConfig,
     SSLConfig sslConfig = new SSLConfig();
     sslConfig.setCiphers(distributionConfig.getSSLCiphers());
     sslConfig
-        .setEndpointIdentificationEnabled(distributionConfig.getSSLEndpointIdentificationEnabled());
+        .setEndpointIdentificationEnabled(distributionConfig.getSSLEndPointIdentificationEnabled());
```

Comment:
distributedConfig.getSSLEndPointIdetificationEnabled would not return null anyway, so as far as SSLConfig is concerned, this is just a simple boolean.